compiler/ras/LimitFile.cpp

```diff
@@ -380,78 +380,108 @@ TR_Debug::scanInlineFilters(FILE * inlineFile, int32_t & lineNumber, TR::Compila
    return !inlineFileError;
    }
 
+/** \brief
+ *     This function opens the inlinefile and adds its entries to a TR::CompilationFilters.
+ *
+ *     An inlinefile is a file containing a list of methods, one per line, which the inliner will limit itself to
+ *     when trying to perform inlining. In other words, only methods from the file can be inlined, but there is no
+ *     guarantee that any of them will be inlined. The format for entry is:
+ *
+ *     + package/Class.method(signature)
```

Comment:
The `+ package/Class.method(signature)` seems very Java-esque and may not relate to the rest of the projects consuming OMR. Perhaps just leaving it as `+ signature` might be more clear? Especially since we have the following API which will return the exact string that should go after the plus sign:

https://github.com/eclipse/omr/blob/8b089c3af5fe41dc700aacf5ade826761687c2cf/compiler/compile/OMRCompilation.hpp#L421

Reply:
Good point.